meta-ibm/meta-witherspoon/recipes-phosphor/webui/phosphor-webui_%.bbappend

```diff
@@ -0,0 +1,2 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+SRC_URI_append_mowgli-oem += " file://0001-Change-IBM-logo-to-OpenBmc-logo.patch"
```

Comment:
Why do you need another machine? Why can't you just use mowgli.conf ?

Reply:
@gtmills Because we need two web interfaces with different logos.
One logo is for IBM and the other one is for openBMC default.
So I thought of adding a machine to meet this requirement.